examples/controlnet/train_controlnet.py

```diff
@@ -784,7 +785,12 @@ def main(args):
         controlnet = ControlNetModel.from_pretrained(args.controlnet_model_name_or_path)
     else:
         logger.info("Initializing controlnet weights from unet")
-        controlnet = ControlNetModel.from_unet(unet)
+        controlnet_kwargs = {}
+        if args.controlnet_conditioning_embedding_out_channels is not None:
+            controlnet_kwargs[
+                "conditioning_embedding_out_channels"
+            ] = args.controlnet_conditioning_embedding_out_channels
+        controlnet = ControlNetModel.from_unet(unet, **controlnet_kwargs)
```

Comment:
optionally passing the new `controlnet_conditioning_embedding_out_channels` argument

Reply:
If possible would be good to avoid this, I don't want to offer too many configuration possibilities here

Reply:
removed!